package.json

```diff
@@ -11,6 +11,7 @@
   "devDependencies": {
     "almond": "^0.3.1",
     "async": "0.9.0",
+    "bower": "^1.4.1",
```

Comment:
ohhh, duh. I'll remove this. I see now that running ``bower install`` was simply wrong so we don't need a top-level dep.